spec/integration/local_spec.rb

```diff
@@ -76,10 +82,13 @@
       end
 
       it 'exposes environment variables during apply' do
-        result = run_cli_json(%w[plan run env_var::get_var] + config_flags)
-        expect(result).not_to include('kind')
-        event = result.first['result']['report']['resource_statuses']['Notify[gettingvar]']['events'].first
-        expect(event).to include('message' => "defined 'message' as 'testing this'")
+        # Uses system ruby instead of bolt's ruby
```

Comment:
It seems like we either need to move this test out of the block where `uri` is set to two nodes or actually make assertions about all the results.

Reply:
I think the test is actually testing whether environment variables are available in an apply block. The comment was misleading (it's what I thought was happening and I forgot to update). In reality I think because it *isn't* getting set to Bolt's ruby because the hostname isn't localhost, it's failing. I think we can safely test statement on just `localhost` and re-set the URI for that test, or we can move it out and assert that it errors. I don't see too much point in asserting an error though, since we're not actually testing that here and have already verified that the interpreter configuration is picked up correctly in the `inventory_spec` test.